scripts/core/editor3/reducers/toolbar.jsx

```diff
@@ -169,6 +171,35 @@ const updateImage = (state, {entityKey, media}) => {
     return onChange(state, newState, entityDataHasChanged);
 };
 
+/**
+ * @ngdoc method
+ * @name removeBlock
+ * @param {Object} data Contains the key from of block to remove
+ * @description Removes block from editor
+ */
+const removeBlock = (state, {blockKey}) => {
+    const {editorState} = state;
+    const contentState = editorState.getCurrentContent();
+
+    const afterKey = contentState.getKeyAfter(blockKey);
+    const targetRange = new SelectionState({
+        anchorKey: blockKey,
+        anchorOffset: 0,
+        focusKey: afterKey,
+        focusOffset: 0
+    });
+    let newContentState = Modifier.setBlockType(
```

Comment:
what does it do?

Reply:
@petrjasek it just gets the selection state for that block and then sets that range to unstyled. After that it removes the block with the selection range ([source](https://github.com/draft-js-plugins/draft-js-plugins/blob/master/draft-js-drag-n-drop-plugin/src/modifiers/removeBlock.js#L30))

Reply:
why it sets the range to unstyled?